--- trunk/simpleclient/tkphone/Makefile.am	2007/07/02 20:41:19	1051
+++ trunk/simpleclient/tkphone/Makefile.am	2008/07/01 15:21:20	1435
@@ -1,5 +1,5 @@
 # Should support LINUX MACOSX WIN32 SOLARIS FREEBSD
-bin_SCRIPTS = tkphone
+bin_SCRIPTS = tkiaxphone
 
 iaxcli_PROGRAMS=iaxcli
 iaxcli_SOURCES = iaxcli.c tones.c tones.h
@@ -9,9 +9,9 @@
 iaxcli_SCRIPTS = tkiaxphone
 dist_iaxcli_DATA = monitor.ui monitor.ui.tcl phone.ui phone.ui.tcl pref.ui pref.ui.tcl
 
-LDADD = $(top_builddir)/lib/libiaxclient.la $(GTK_LIBS) $(GDK_LIBS)
+LDADD = $(top_builddir)/lib/libiaxclient.la $(GTK_LIBS) $(GDK_LIBS) $(PORTAUDIO_LIBS) $(SPEEX_LIBS)
 AM_CPPFLAGS = -I$(top_srcdir)/lib -DUSE_HOTKEY=0
-AM_CFLAGS = $(GTK_CFLAGS) $(GDK2_CFLAGS)
+AM_CFLAGS = $(GTK_CFLAGS) $(GDK2_CFLAGS) $(PORTAUDIO_CFLAGS) $(SPEEX_CFLAGS)
 
 if ! WIN32
   AM_CPPFLAGS += -DPOSIXSLEEP
--- trunk/simpleclient/tkphone/iaxcli.c	2007/05/07 20:11:10	964
+++ trunk/simpleclient/tkphone/iaxcli.c	2008/07/01 15:21:20	1435
@@ -413,7 +413,7 @@
 #endif
 
     atexit(iaxc_shutdown); /* activate the exit handler */
-    if (iaxc_initialize(AUDIO_INTERNAL_PA,1)) {
+    if (iaxc_initialize(1)) {
 	fatal_error("cannot initialize iaxclient!");
     }
 

